src/app/parcels/ActionBar/ActionAndStatusBar.cy.tsx

```diff
@@ -36,6 +36,7 @@ describe("Parcels - Action Bar", () => {
             },
             voucherNumber: "123456789",
             createdAt: new Date("2023-12-31T12:00:00+00:00"),
+            clientIsActive: true,
```

Comment:
These component tests are deprecated because of this and currently failing so I wouldn't be able to test if it worked - see this Slack message https://softwire.slack.com/archives/C05E17YJQN4/p1710868145285059

But I totally agree it would be a good idea to include if we used the component tests